Firebase/Auth/Source/Public/FIREmailAuthProvider.h

```diff
@@ -26,7 +26,7 @@ NS_ASSUME_NONNULL_BEGIN
 extern NSString *const FIREmailAuthProviderID NS_SWIFT_NAME(EmailAuthProviderID);
 
 /**
-    @brief please use `FIREmailAuthProviderID` instead.
+    @brief please use `EmailAuthProviderID` instead.
```

Comment:
change to : "Please use `FIREmailAuthProviderID` for Objective-C or `EmailAuthProviderID` for Swift.